sugondat-chain/dynamic-fee-adjustment/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "dynamic-fee-adjustment"
```

Comment:
How about `sugondat-length-fee-adjustment`?

Reply:
OK, for now, I used `pallet-sugondat-length-fee-adjustment`. The only thing is that the pallet currently implements both length and weight to the fee. So, `pallet-sugondat-dynamic-fee-adjustment` is probably more general?